### PR TITLE
Add and run prettier

### PR DESCRIPTION
**Pull request by PorgJuice**
I have some more substantial changes I'd like to make, but wanted to run this first. Prettier is the easiest way to keep coding style consistent, but I'm unopinionated as to what the enforced styles should be. I've just run prettier with the default configuration. Happy to apply a specific configuration.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -62,3 +62,9 @@ buck-out/
 *.jsbundle
 electron/
 public/favicon*
+
+# Spacemacs
+**/.#*
```
Comment (PorgJuice) [inline .gitignore:67]: The flycheck plugin for spacemacs drops a bunch of turds every time it analyzes a file. This just excludes those.

File: src/actions/ActionTypes.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ export const CONNECT_SUCCESSFUL = "CONNECT_SUCCESSFUL";
 export const CONNECT_FAILED = "CONNECT_FAILED";
 export const LOGIN_SUCCESSFUL = "LOGIN_SUCCESSFUL";
 export const LOGIN_FAILED = "LOGIN_FAILED";
-export const SOMETHING_HAPPENED_SUCCESSFULLY = "SOMETHING_HAPPENED_SUCCESSFULLY";
+export const SOMETHING_HAPPENED_SUCCESSFULLY =
```
Comment (EraYaN) [inline src/actions/ActionTypes.js]: Please do not enforce such a short line width.

**Review by EraYaN — CHANGES_REQUESTED** [review]
Also please respect the `.editorconfig` file in the main repositories too, so indentations are 4 spaces across the repositories. There is no max line length enforced, gone are the days of people with 800x600 screens, so there is no need as long are people don't go overboard.

[This one](https://github.com/jellyfin/jellyfin-web/blob/master/.editorconfig) and [this one](https://github.com/jellyfin/jellyfin/blob/master/.editorconfig)

File: src/actions/ActionTypes.js (hunk 2 above)
Comment (PorgJuice) [reply src/actions/ActionTypes.js]: Fixed.

**Review by EraYaN — APPROVED** [review]

**Pull request merged**